specification/datafactory/resource-manager/Microsoft.DataFactoryV1Extended/2017-08-10-preview/swagger/datafactory.json

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
This file should appear one level up. We shouldn't have an extra folder `swagger`.